products/lineageos.md

```diff
@@ -16,25 +16,45 @@ releases:
     eol: false
     releaseDate: 2022-12-31
     link: https://lineageos.org/Changelog-27/
+    androidVersion: 13
 
 -   releaseCycle: "19"
     eol: false
     releaseDate: 2022-04-26
     link: https://lineageos.org/Changelog-26/
+    androidVersion: 12
 
 -   releaseCycle: "18.1"
     eol: false
     releaseDate: 2021-04-01
     link: https://lineageos.org/Changelog-25/
+    androidVersion: 11
 
 -   releaseCycle: "17.1"
     eol: 2022-04-26
     releaseDate: 2020-02-01
     link: https://lineageos.org/Changelog-24/
+    androidVersion: 10
 
 -   releaseCycle: "16.0"
     eol: 2021-02-16
     releaseDate: 2019-03-01
+    androidVersion: 9
+
+-   releaseCycle: "15.1"
+    eol: true
+    releaseDate: 2018-02-26
+    androidVersion: 8.1
+
+-   releaseCycle: "14.1"
+    eol: true
+    releaseDate: 2017-01-22
```

Comment:
Where can I confirm this date ? Can you add, as a comment, a link to a lineageos.org related page confirming this ?